src/ACFFields.php

```diff
@@ -0,0 +1,153 @@
+<?php
+/**
+ * PHP version 5.6
+ *
+ * @package wpml-to-polylang
+ */
+
+namespace WP_Syntex\WPML_To_Polylang;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Handles the ACF fields.
+ *
+ * @since 0.6
+ */
+class ACFFields extends AbstractSteppable {
+	const ACFML_KEY = 'wpml_cf_preferences';
+	const TYPES     = [
+		0 => 'none', // WPML_IGNORE_CUSTOM_FIELD.
+		1 => 'copy', // WPML_COPY_CUSTOM_FIELD.
+		2 => 'translate', // WPML_TRANSLATE_CUSTOM_FIELD.
+		3 => 'copy_once', // WPML_COPY_ONCE_CUSTOM_FIELD.
+	];
+
+	/**
+	 * Batch size.
+	 *
+	 * @var int
+	 */
+	protected $batch_size;
+
+	/**
+	 * Constructor.
+	 *
+	 * @since 0.6
+	 */
+	public function __construct() {
+		$this->batch_size = absint( WPML_TO_POLYLANG_QUERY_BATCH_SIZE / 100 ); // 50 by default, to limit the size of the UPDATE query.
+	}
```

Comment:
Is it useful to have a property since it's used in only one method?

Reply:
Changed in [85c419b](https://github.com/polylang/wpml-to-polylang/pull/25/commits/85c419b376580f43f9485aedb95d803434f666d9).